src/support/kenniscloud_schema.erl

```diff
@@ -56,6 +56,18 @@ manage_schema({upgrade, 18}, _Context) ->
                     {acl_collaboration_group, acl_collaboration_group}
             ]}
         ]
+    };
+manage_schema({upgrade, 19}, _Context) ->
+    #datamodel{
+        predicates=[
+            {hasextra_rsc, [
+                {title, {trans, [{nl, <<"Extra: Resources">>},
+                                 {en, <<"Extra: Resources">>}]}},
+                {language, [en,nl]}
+            ], [
+                {acl_collaboration_group, reference}
+            ]}
+        ]
```

Comment:
The definition of this predicate in `get_prod_data` should also be updated to match this one (so that installing from scratch has the same result as upgrading the site).